<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intérprete de sueños</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next_e_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Episodio Siguiente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_e_btn">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>560</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Episodio Anterior</string>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="plotConsumo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>951</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadFile">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>590</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
